cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/Conversion.java

```diff
@@ -271,9 +275,54 @@ public Network convert(ReportNode reportNode) {
         }
 
         CgmesReports.importedCgmesNetworkReport(reportNode, network.getId());
+
+        updateWithAllInputs(network, context, reportNode);
+
         return network;
     }
 
+    private void updateWithAllInputs(Network network, Context convertContext, ReportNode reportNode) {
+        if (!sshOrSvIsIncludedInCgmesModel(this.cgmes)) {
+            return;
+        }
+        this.cgmes.invalidateCaches();
+        this.cgmes.setQueryCatalog("-update");
+        Context updateContext = createUpdateContext(network, reportNode);
+
+        // add processes to create new equipment using update data (ssh and sv data)
+
+        update(network, updateContext, reportNode);
+    }
+
+    // TODO Remove CIM14 support after PR #3375 (Drop support for CIM14) has been merged into the main branch
+    private static boolean sshOrSvIsIncludedInCgmesModel(CgmesModel cgmes) {
+        return cgmes.version().contains("CIM14")
+                || cgmes.fullModels().stream()
+                .map(fullModel -> fullModel.getId("profileList"))
+                .anyMatch(profileList -> profileList.contains("SteadyStateHypothesis") || profileList.contains("StateVariables"));
+    }
+
+    public void update(Network network, ReportNode reportNode) {
+        Objects.requireNonNull(network);
+        Objects.requireNonNull(reportNode);
+        Context updateContext = createUpdateContext(network, reportNode);
+        update(network, updateContext, reportNode);
+    }
+
+    private void update(Network network, Context updateContext, ReportNode reportNode) {
+        // FIXME(Luma) Inspect the contents of the loaded data
```

Comment:
Can you also delete this FIXME? I think you missed it.

Reply:
Done